code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -110,12 +110,20 @@
 		if(I_HARM)
 			//Vampire code
 			if(M.mind && M.mind.vampire && (M.mind in ticker.mode.vampires) && !M.mind.vampire.draining && M.zone_sel && M.zone_sel.selecting == "head" && src != M)
+				if(species && species.flags & NO_BLOOD)//why this hell were we never checkinf for this?
+					to_chat(M, "<span class='warning'>They have no blood!</span>")
 				if((head && (head.flags & HEADCOVERSMOUTH)) || (wear_mask && (wear_mask.flags & MASKCOVERSMOUTH)))
 					to_chat(M, "<span class='warning'>Remove their mask!</span>")
 					return
 				if((M.head && (M.head.flags & HEADCOVERSMOUTH)) || (M.wear_mask && (M.wear_mask.flags & MASKCOVERSMOUTH)))
-					to_chat(M, "<span class='warning'>Remove your mask!</span>")
-					return
+					if(M.get_species() == "Vox")

```

Comment:
This way of coding it makes it skip all other checks. Vox can suck skeletons dry. 😏